.github/workflows/e2e-tests.yaml

```diff
@@ -28,9 +28,20 @@ jobs:
       - operator-e2e
     outputs:
       cluster_status: ${{ steps.run_e2e_test.outputs.cluster_status }}
+      job_id: ${{ steps.get-job-id.outputs.job_id }}
     steps:
       - name: Checkout
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: Get Job ID from GH API
+        id: get-job-id
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gh
+          jobs=$(gh api repos/${{ github.repository }}/actions/runs/${{ github.run_id}}/attempts/${{ github.run_attempt }}/jobs)
+          job_id=$(echo $jobs | jq -r '.jobs[] | select(.runner_name=="${{ runner.name }}") | .id')
```

Comment:
Have you tried retrieving the job ID via `${{ job.check_run_id }}`?

Should be working already, but hasn't been documented yet unfortunately: https://github.com/orgs/community/discussions/8945#discussioncomment-14374985

Reply:
fixed in f123befa9